uk.co.compucorp.civicrm.tasksassignments/xml/activity_custom_fields.xml

```diff
@@ -53,7 +54,70 @@
       <note_columns>60</note_columns>
       <note_rows>4</note_rows>
       <column_name>clone_date</column_name>
+      <in_selector>0</in_selector>
+      <custom_group_name>Activity_Custom_Fields</custom_group_name>
+    </CustomField>
+    <CustomField>
+      <name>Valid_From</name>
+      <label>Valid From</label>
+      <data_type>Date</data_type>
+      <html_type>Select Date</html_type>
+      <is_required>0</is_required>
+      <is_searchable>1</is_searchable>
+      <is_search_range>1</is_search_range>
+      <weight>19</weight>
```

Comment:
No, just I used the autogenerated weights from the exported xml. I'll change them to sequential.